src/main/webapp/js/user-page-loader.js

```diff
@@ -86,10 +87,80 @@ function fetchMessages() {
       });
 }
 
+function createMap(){
```

Comment:
This function looks similar to a function in #40 in `map-loader.js`, would it be possible to reuse anything from there?